Assets/Scripts/DeviceCommunicator.cs

```diff
@@ -10,18 +8,18 @@ public class DeviceCommunicator : MonoBehaviour {
 	[Inject]
 	CodeExecutor codeExecutor;
 
-	public void Listen(string protobufInstructionsString) 
+	public void Listen(string protobufInstructionsString)
 	{
 		byte[] protobufInstructions = Encoding.Default.GetBytes (protobufInstructionsString);
 		Code code = Code.Parser.ParseFrom (protobufInstructions);
 
 		codeExecutor.Run (code);
 	}
 
-	[DllImport ("__Internal")]
-	private extern static void endLevel(LevelCompleteStatus status);
+	[System.Runtime.InteropServices.DllImport("__Internal")]
```

Comment:
I changed to this approach after reading [this article](https://github.com/jiulongw/swift-unity/issues/14) which was published mid-November laster year. It seems to do what we want, but in a much more straight forward way.